habitat-hitl/habitat_hitl/_internal/networking/networking_process.py

```diff
@@ -518,8 +518,24 @@ async def http_handler(request):
         # print(f"Returned availability HTTP code {code}")
         return aiohttp.web.Response(status=code)
 
+    async def get_status(request):
+        return aiohttp.web.json_response(
+            {
+                "accepting_users": network_mgr.is_server_available(),
+                "user_count": len(network_mgr._user_slots),
+            },
+            text=None,
+            body=None,
+            status=200,
```

Comment:
Here again, it's confusing for this to be called "get_status" when it also happens to return a status code of 200 regardless of the... "status".